Python/shape-detection/Shape_Detection.py

```diff
@@ -0,0 +1,201 @@
+                shapes["Square"].append(cy)        
+            elif(w<(w_img-10) and areaa>=0.99*w*h):
+                shapes["Rectangle"]=[]
+                ar=area(cnt)
+                cx,cy=centroids(cnt)
+                color = image[cy,cx]
+                blue,green,red=color
+
+                m=max(blue,green,red)
+                if color[0]==m:
+                    col="blue"
+                elif color[1]==m:
+                    col= "green"
+                else:
+                    col="red"
+                shapes["Rectangle"].append(col)    
+                shapes["Rectangle"].append(ar)
+                shapes["Rectangle"].append(cx)
+                shapes["Rectangle"].append(cy)
+
+            elif(w<(w_img-10)):
+                shapes["Rhombus"]=[]
+                ar=area(cnt)
+                cx,cy=centroids(cnt)
+                color = image[cy,cx]
+                blue,green,red=color
+
+                m=max(blue,green,red)
+                if color[0]==m:
+                    col="blue"
+                elif color[1]==m:
+                    col= "green"
+                else:
+                    col="red"
+                shapes["Rhombus"].append(col)    
+                shapes["Rhombus"].append(ar)
+                shapes["Rhombus"].append(cx)
+                shapes["Rhombus"].append(cy)
+           
+            
+        elif len(approx) == 5:
+            shapes["Pentagon"]=[]
+            ar=area(cnt)
+            cx,cy=centroids(cnt)
+            color = image[cy,cx]
+            blue,green,red=color
+
+            m=max(blue,green,red)
+            if color[0]==m:
+                col="blue"
+            elif color[1]==m:
+                col= "green"
+            else:
+                col="red"
+            shapes["Pentagon"].append(col)
+            shapes["Pentagon"].append(ar)
+            shapes["Pentagon"].append(cx)
+            shapes["Pentagon"].append(cy)
+        elif len(approx) == 6:
+            shapes["Hexagon"]=[]
+            ar=area(cnt)
+            cx,cy=centroids(cnt)
+            color = image[cy,cx]
+            blue,green,red=color
+
+            m=max(blue,green,red)
+            if color[0]==m:
+                col="blue"
+            elif color[1]==m:
+                col= "green"
+            else:
+                col="red"
+            shapes["Hexagon"].append(col)
+            shapes["Hexagon"].append(ar)
+            shapes["Hexagon"].append(cx)
+            shapes["Hexagon"].append(cy)    
+        elif 6 < len(approx) < 15:
+            shapes["Ellipse"]=[]
+            ar=area(cnt)
+            cx,cy=centroids(cnt)
+            color = image[cy,cx]
+            blue,green,red=color
+
+            m=max(blue,green,red)
+            if color[0]==m:
+                col="blue"
+            elif color[1]==m:
+                col= "green"
+            else:
+                col="red"
+            shapes["Ellpise"].append(col)
```

Comment:
Traceback (most recent call last):
  File "shape.py", line 174, in <module>
    shapes["Ellpise"].append(col)
KeyError: 'Ellpise'

I am getting this error in line no. 174